portworx-install-docker-standalone-2-6/step3.md

```diff
@@ -2,7 +2,7 @@
 Install the Bundle:
 
 
-`latest_stable=$(curl -fsSL "https://install.portworx.com/?type=dock&stork=false" | awk '/image: / {print $2}')`{{execute node01}}
+`latest_stable=$(curl -fsSL "https://install.portworx.com/2.6/?type=dock&stork=false" | awk '/image: / {print $2}')`{{execute node01}}
```

Comment:
doesnt really matter here since we dont care about Kubernetes in these labs. fine to keep though

Reply:
This did make a difference; I could not get a successful pull of our images at all without using 2.6 tree from install.portworx.com....